bin/preview.py

```diff
@@ -21,7 +20,7 @@
 @APP.route(DOCS_ROUTE, methods=["GET"])
 @APP.route(DOCS_ROUTE + "<path:path>", methods=["GET"], defaults={"path": None})
 @APP.route(DOCS_ROUTE + "<path:path>", methods=["GET"])
-def static_proxy (path=""):
+def static_proxy (path=""): # pylint: disable=C0116
```

Comment:
```suggestion
def static_proxy (path=""):
    """Serve static files from the /site directory."""
```

Reply:
Thank you! I recognize what the functions do, but didn't want to provide docstrings for them without properly understanding what files they host locally.